python_modules/libraries/dagster-dbt/dagster_dbt/cloud/resources.py

```diff
@@ -713,6 +714,7 @@ def my_dbt_cloud_job():
 # ------------------
 
 DAGSTER_ADHOC_PREFIX = "DAGSTER_ADHOC_JOB__"
+LIST_JOBS_INDIVIDUAL_REQUEST_LIMIT = 100
```

Comment:
Nit: make configurable via env var

Reply:
This shouldn't be here, it's another restack conflict gone wrong - it was introduced in `clients.py` in #28147. Will do in a PR on its own